README.md

```diff
@@ -410,3 +410,18 @@ android {
   }
 }
 ```
+
+To publish the library to a local Maven repository for test purposes run:
+-------
+
+Gradle:
+```bash
+./gradlew publishToMavenLocal -x test
+```
+
+Maven:
+```bash
+mvn install -DskipTests
+```
+
+`-x test` in Gradle and `-DskipTests` in Maven is used to bypass the outdated failing tests.
```

Comment:
@sealedtx This PR fixed several related tests automatically. 

But FYI, another batch of tests is failing now due to various reasons (like video no longer exists, or playlist is outdated, etc), so to test it locally, I added an instruction to deliberately skip the failing tests